homeassistant/components/watson_tts/tts.py

```diff
@@ -99,6 +102,7 @@ def get_engine(hass, config):
     supported_languages = list({s[:5] for s in SUPPORTED_VOICES})
     default_voice = config[CONF_VOICE]
     output_format = config[CONF_OUTPUT_FORMAT]
+    service.set_default_headers({'x-watson-learning-opt-out': config[CONF_TELEMETRY]})
```

Comment:
I think true/false is being swapped here.

Reply:
I simply disabled it on my side with custom component, but i though i might add a check
and your are right, default should be true if disable by default ... i think i changed my mind about what should be default to many time XD

Reply:
should be better